src/xtgeo/grid3d/_gridprop_import_eclrun.py

```diff
@@ -568,3 +565,35 @@ def import_gridprops_from_restart_file(
     if close:
         filehandle.close()
     return read_properties
+
+
+def import_gridprop_from_init(pfile, name, grid, fracture=False):
```

Comment:
I find it a bit hard to parse that `import_gridprop_from_init` calls `import_gridprop_from_init_file`.
Do we need both?

Reply:
This is indeed some bad naming, we need both as we will need to have an entry point which does the search both when we are getting just one grid property and when we are getting any matching a search criteria. I guess some wording involving `search` or `collect` might help.

Reply:
I tried a rename to collect in case that helps.